lib/crowbar/client/command/backup/upload.rb

```diff
@@ -19,6 +19,17 @@ module Command
       module Backup
         class Upload < Base
           def request
+            args.file = case args.file
+            when "-"
+              stdin.to_io
+            when File
```

Comment:
Style/CaseIndentation: Indent `when` as deep as `case`. (https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)